pkg/kubelet/cm/qos_container_manager_linux.go

```diff
@@ -139,7 +139,12 @@ func (m *qosContainerManagerImpl) Start(getNodeAllocatable func() v1.ResourceLis
 }
 
 func (m *qosContainerManagerImpl) setCPUCgroupConfig(configs map[v1.PodQOSClass]*CgroupConfig) error {
-	pods := m.activePods()
+	var pods []*v1.Pod
```

Comment:
This changes was introduced in 1.6 and still on master today.
Running the kubelet with `--runonce` give you this:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xdfefeb]

goroutine 171 [running]:
panic(0x30b3260, 0xc4200120f0)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
k8s.io/kubernetes/pkg/kubelet/cm.(*qosContainerManagerImpl).setCPUCgroupConfig(0xc4200b7180, 0xc420cfe300, 0xc420cc9228, 0xc420cc9220)
	/home/jb/go/src/github.com/JulienBalestra/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubelet/cm/qos_container_manager_linux.go:142 +0x5b
k8s.io/kubernetes/pkg/kubelet/cm.(*qosContainerManagerImpl).UpdateCgroups(0xc4200b7180, 0x0, 0x0)
	/home/jb/go/src/github.com/JulienBalestra/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubelet/cm/qos_container_manager_linux.go:268 +0x28c
k8s.io/kubernetes/pkg/kubelet/cm.(*containerManagerImpl).UpdateQOSCgroups(0xc42024e960, 0xc4207a4f00, 0xc420cbab00)
	/home/jb/go/src/github.com/JulienBalestra/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubelet/cm/container_manager_linux.go:480 +0x3a
k8s.io/kubernetes/pkg/kubelet.(*Kubelet).syncPod(0xc42079c000, 0x0, 0xc4207a4f00, 0x1, 0xc420b10d90, 0x0, 0x7, 0xc420b10d90)
	/home/jb/go/src/github.com/JulienBalestra/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubelet/kubelet.go:1473 +0x1695
k8s.io/kubernetes/pkg/kubelet.(*Kubelet).runPod(0xc42079c000, 0xc4207a4f00, 0x3b9aca00, 0x0, 0x0)
	/home/jb/go/src/github.com/JulienBalestra/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubelet/runonce.go:139 +0x411
k8s.io/kubernetes/pkg/kubelet.(*Kubelet).runOnce.func1(0xc42079c000, 0x3b9aca00, 0xc420bf25a0, 0xc4207a4f00)
	/home/jb/go/src/github.com/JulienBalestra/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubelet/runonce.go:87 +0x43
created by k8s.io/kubernetes/pkg/kubelet.(*Kubelet).runOnce
	/home/jb/go/src/github.com/JulienBalestra/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubelet/runonce.go:89 +0x1dc
```